front/src/components/header/subcomponents/LoginModal.tsx

```diff
@@ -245,48 +300,49 @@ const LoginModal = ({ disclosureProps, setIsLoggedIn, unregisteredClearFollowedS
           </ModalBody>
 
           <ModalFooter>
-            <Grid gap="20px" gridTemplateColumns="1fr 3fr">
-              <Box textAlign="left">
-                {/* The below button is visible in first & second formOption only and toggle between them (i.e. Login, Send One Time Code)
-                    when formOption = 0 : (0 + 1) % 2 which is 1
-                    when formOption = 1 : (1 + 1) % 2 which is 0
-                */}
-                {(formOption === 0 || formOption === 1) && (
-                  <Button
-                    _active={{
-                      borderColor: 'none',
-                    }}
-                    _focus={{
-                      borderColor: 'none',
-                    }}
-                    color="#659BC7"
-                    fontSize="0.88rem"
-                    onClick={() => setFormOption((formOption + 1) % 2)}
-                    pt="0.75rem"
-                    variant="link"
-                  >
-                    {(formOption === 0 && 'Forgot Password?') ||
-                      (formOption === 1 && (
-                        <>
-                          <Box as={TiArrowBack} size="18px" />
-                          Go back
```

Comment:
I think there might have been a merge conflict here - this version of the Back button with `TiArrowBack` is the newer version and should be included, but below the version of `< Back to Login` is included which is older

Reply:
I updated my forked repository and cloned the updated one, but still it didn't reflect the changes. Weird. Alright, I'm updating the code right now.